.gitignore

```diff
@@ -4,11 +4,16 @@ Release
 .idea
 *.iml
 build_android/build
+build_android/bin
```

Comment:
With the current build setup, the `bin`, `gen`, `libs` and `obj` folders shouldn't be generated anymore.